app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    comand, source_file, destination = command.split()
+
+    if comand == "mv" and source_file != destination:
+        if destination.endswith("/"):
```

Comment:
you should  to perform path-related operations in a platform-independent way, you can use the os.path module. In this case, you can use os.path.sep to obtain the platform-specific path separator and then check if the destination string ends with that separator.